lib/material/extra/calendar/calendar.flow

```diff
@@ -928,17 +933,16 @@ dayEventsPopup(
 	close = make(false);
 	dayEvents = make(dayEventsStatic);
 
-	curTime = stamp2time(timestamp());
-	dayNumber = curDate.day;
-	monthNumber = curDate.month;
-
 	roundTime = \time -> {
-		roundInterval = 30;
-		roundedMinutes = (time.min / roundInterval + 1) * roundInterval;
-		flooredTime = Time(time with min = 0, sec = 0);
-		stamp2time(time2stamp(flooredTime) + min2ms(roundedMinutes));
+		if (i2b(time.min) || i2b(time.sec)) {
```

Comment:
What if time will be with 0 min and 0 sec without CalendarGetHourOnEmptyCell() style? Not likely but could happen. Then it won't work as it should.
Maybe just check styles for CalendarGetHourOnEmptyCell() style?

Reply:
I've added check for getHourOnClick

Reply:
Shouldn't it be just ```if (!getHourOnClick) {```?